src/pages/components/tooltip-modal.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Tooltip Modal
+description: >
+  A Tooltip modal is a small overlay that opens automatically or through user-action, and displays contextual information for its paired element.
```

Comment:
A Tooltip modal provides contextual information about a paired element. It either opens automatically or through user action.

Reply:
Updated.